packages/snaps-controllers/src/services/iframe/IframeExecutionService.test.ts

```diff
@@ -18,6 +18,50 @@ import {
 
 const iframeUrl = new URL(`http://localhost:${serverPort}`);
 
+const MOCK_BLOCK_NUM = '0xa70e75';
+
+const createService = () => {
```

Comment:
This seems to be similar to `AbstractExecutionService.test.ts`. Can we re-use this somehow?

Reply:
Each of them initialize a different kind of service though. Not sure how to re-use efficiently 🤔